app/routes/matador.tsx

```diff
@@ -55,8 +55,12 @@ export default function MatadorLayout() {
   );
 }
 
-export function ErrorBoundary() {
+export function CatchBoundary() {
   const caught = useCatch();
 
   return <ErrorFallback error={caught} />;
 }
+
+export function ErrorBoundary({ error }: { error: Error }) {
```

Comment:
Please declare a props type for `ErrorBOundary`

Reply:
Fixed with f10a13c4bcd4c0af9f370d210dfce95f105179ce